src/TcUnit.TestAdapter/Models/SystemService.cs

```diff
@@ -82,7 +97,7 @@ public void SwitchRuntimeState(AdsState newState)
                             {
                                 throw;
                             }
-                            adsClient.Connect(10000);
+                            adsClient.Connect(amsNetId, (int)AmsPort.SystemService);
```

Comment:
The ADS client doesn't remember which net ID it was connected to, so this was trying to reconnect to the local AmsNetID.